src/glyphnote.ts

```diff
@@ -1,15 +1,20 @@
 // [VexFlow](http://vexflow.com) - Copyright (c) Mohit Muthanna 2010.
 
+import { Vex } from './vex';
 import { BoundingBox } from './boundingbox';
 import { Glyph } from './glyph';
 import { Note, NoteStruct } from './note';
+import { ModifierContext } from './modifiercontext';
+import { ModifierContextState } from './modifiercontext';
```

Comment:
This line can be deleted, `ModifierContextState` is not used.

Reply:
done.  Didn't it used to throw an error when you did that?

Reply:
The line is still there